FMPFeedbackForm/Sources/Extensions/NSColor+FMPExtensions.m

```diff
@@ -70,7 +70,11 @@ + (instancetype)fmp_colorNamed:(NSString *)colorName fallbackColor:(NSColor *)fa
 {
     if (@available(macOS 10.13, *))
     {
+#if SWIFT_PACKAGE
+        return [NSColor colorNamed:colorName bundle:SWIFTPM_MODULE_BUNDLE];
+#else
         return [NSColor colorNamed:colorName bundle:FMPBundleHelper.currentBundle];
```

Comment:
How about putting this into the `FMPBundleHelper.currentBunde` instead?

Reply:
Good catch! Refactored bundle selection into `currentBundle`.